ts/App.tsx

```diff
@@ -14,16 +14,20 @@ import RootContainer from "./RootContainer";
 
 // Inizialize Mixpanel and configure the global js error handler
 Mixpanel.sharedInstanceWithToken(config.mixpanelToken);
+
+// Configure the global error handler
 configureErrorHandler()
   // tslint:disable-next-line:no-empty
   .then(() => {})
```

Comment:
actually we need to think what we do here, we can probably send something via mixpanel

Reply:
I created a story time ago to implement the error handling in a better way https://www.pivotaltracker.com/n/projects/2048617/stories/155392873